app/src/main/res/layout/pin_password_activity.xml

```diff
@@ -95,7 +95,7 @@
             app:layout_constraintTop_toBottomOf="@id/hello_text_view" />
 
           <androidx.constraintlayout.widget.ConstraintLayout
-            android:layout_width="@dimen/pin_password_activity_input_with_icon_width"
+            android:layout_width="match_parent"
```

Comment:
Is this change actually correct to make? A solution that doesn't require changing dimensions would be collapsing the nested ConstraintLayouts (which seems like the more serious issue here as it is both a performance issue and complex bugs like the one this PR exposed).

Reply:
I have reverted back to dimen and inside dimens.xml I have used -1px which is equivalent to match parent. I have referred this [stackoverflow](https://stackoverflow.com/a/37971851/12314934) answer.